pymc/distributions/continuous.py

```diff
@@ -3669,6 +3669,13 @@ def dist(cls, x_points, pdf_points, *args, **kwargs):
 
         return super().dist([x_points, pdf_points, cdf_points], **kwargs)
 
+    def get_moment(rv, size, x_points, pdf_points):
+        moment = at.mul(x_points, pdf_points)
+        if not rv_size_is_none(size):
+            moment = at.full(size, moment)
+
```

Comment:
Are `x_points` and `pdf_points` 1D vectors? [This line](https://github.com/pymc-devs/pymc/blob/main/pymc/distributions/continuous.py#L3658) makes me think that they are

Reply:
Yes they have to be. We should assert that in the RV.make_node method